browser/src/layer/vector/SVGGroup.js

```diff
@@ -165,7 +165,7 @@ L.SVGGroup = L.Layer.extend({
 	_onDragStart: function(evt) {
 		if (!this._map || !this._dragShapePresent || !this.dragging)
 			return;
-		if (this._map._docLayer._cursorMarker && this._map._docLayer._cursorMarker.isVisible())
+		if (this._map._docLayer._cursorMarker && $('.leaflet-pane.leaflet-map-pane').css('cursor') === 'text')
```

Comment:
This query can fail and cause error.
Also please don't use queries when possible. Can't we get object by getting the section reference and using it's member?